examples/language-modeling/README.md

```diff
@@ -615,6 +615,8 @@ python3 ../gaudi_spawn.py --use_deepspeed  --world_size 8  run_lora_clm.py \
 
   > The following command requires Habana DeepSpeed 1.13.0 or later.
 
+  > The `--flash_attention_fp8` flag can be used for applying Flash Attention in FP8 precision. Note that this feature requires Gaudi 3 accelerators!
```

Comment:
Ideally we need to test this use-case on G3 to check if training completes without problems and also to quantify if we are getting any performance gains from enabling flash_attention_fp8.  Let's discuss this further.

Reply:
Removed this information